### PR TITLE
Store the release group's title.

### PR DESCRIPTION
**Pull request by djl**
beets does not currently store the "canonical" release group title, only each release's title and disambiguation. Oddly, beets does store the release group disambiguation.

Without this there is no way to determine a release group's title without separately querying MusicBrainz with `mb_releasegroupid`.

## To Do

- [x] Documentation. (If you've add a new command-line flag, for example, find the appropriate page under `docs/` to describe it.)
- [x] Changelog. (Add an entry to `docs/changelog.rst` near the top of the document.)
- [ ] Tests. (Encouraged but not strictly required.)

### CODE REVIEW

**Comment by sampsyo** [comment]
Seems cool overall! But maybe we should just call the field `release_group_title` or similar… since it's not an MBID, it's not really all that MusicBrainz-specific (and you can imagine other data sources hypothetically supplying equivalent data). That would also avoid the need to do any special handling in `apply_metadata`; we could use the same field name all the way through.

**Comment by djl** [comment]
You're totally right, I'm so used to thinking in MB terms. Changed the code and changelog to use `release_group_title` instead.

File: beets/autotag/__init__.py [hunk 1]
```diff
@@ -40,6 +40,7 @@
     'album': (
         'va',
         'releasegroup_id',
+        'release_group_title',
```
Comment (sampsyo) [inline beets/autotag/__init__.py:43]: I think we don't need this—if we leave this off, this data will get applied automatically.

File: beets/autotag/__init__.py [hunk 2]
```diff
@@ -176,6 +177,7 @@ def apply_metadata(album_info: AlbumInfo, mapping: Mapping[Item, TrackInfo]):
             item.mb_artistid = album_info.artist_id
         item.mb_albumartistid = album_info.artist_id
         item.mb_releasegroupid = album_info.releasegroup_id
+        item.release_group_title = album_info.release_group_title
```
Comment (sampsyo) [inline beets/autotag/__init__.py:180]: Regarding the above: I think we can just delete this line, and the `for field, value in album_info.items():` loop below should have the same effect.

File: docs/changelog.rst [hunk 3]
```diff
@@ -88,6 +88,8 @@ New features:
 * :doc:`/plugins/convert`: Add support for generating m3u8 playlists together
   with converted media files.
   :bug:`4373`
+* Add release_group_title field
```
Comment (sampsyo) [inline docs/changelog.rst:91]:
```suggestion
* Fetch the ``release_group_title`` field from MusicBrainz.
```

**Review by sampsyo — COMMENTED** [review]
Looks great! I think we can get away with deleting a couple of lines in the metadata application bit now that we use the same field name consistently.

**Comment by djl** [comment]
Addressed all the above comments.

**Review by sampsyo — APPROVED** [review]
Excellent! Thanks for seeing this through!

**Pull request merged**